src/NServiceBus.Core/OpenTelemetry/Tracing/ActivityTags.cs

```diff
@@ -32,7 +32,7 @@ static class ActivityTags
         public const string PreviousConversationId = "nservicebus.previous_conversation_id";
         public const string MessageIntent = "nservicebus.message_intent";
         public const string TimeToBeReceived = "nservicebus.time_to_be_received";
-        public const string DataBusContentType = "nservicebus.databus.content_type";
+        public const string DataBusConfigContentType = "nservicebus.databusconfig.content_type";
```

Comment:
I think `nservicebus.databus` was considered as a sort of "category" for further potential activity tags that belong to the databus (similar to `nservicebus.saga.[saga_activity_name]`. Not all headers are translated 1:1 to the activity name. I think it would be viable to keep it at the original `"nservicebus.databus.content_type"` value

Reply:
I agree with Tim, it doesn't necessarily need to be a 1/1 translation. I checked the public docs, and we're not mentioning which tags are added. If it were to be changed, this would be a breaking change to the ActivitySource, but we haven't released it yet, so you still have some freedom there...

The question is what we want to convey from the activity tags perspective, a categorization to easily find things, or rather a 1/1 mapping, and I think that question has already been answered since we don't have 1/1 mapping, to begin with.

Reply:
@timbussmann @lailabougria we could keep the OT tag name to what it was, just update the mapping. However, the names would then no longer be the same except for styling.